### PR TITLE
Update notify method to strip escaped HTML characters.

### PR DESCRIPTION
**Pull request by colonelpopcorn**
So, after doing some investigation I found that it was actually the `notify` method in the `RemotePlayerService` class. Not sure why some properties show escaped HTML characters instead of their standard equivalents, and it may be worth looking into why the characters are returned that way from the server. For now, however, this should suffice and allow special characters to be displayed without issue. See the before and after picture below for proof of bug fix, as well.
jelly-fin-before-after

EDIT: This addresses issue #731, btw

### CODE REVIEW

File: app/src/main/java/org/jellyfin/mobile/utils/HtmlUtils.kt [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+import android.text.Html
+
+@Suppress("DEPRECATION")
+fun stripHtmlChars(content: String?): String {
+    return if (android.os.Build.VERSION.SDK_INT >= android.os.Build.VERSION_CODES.N) {
+        Html.fromHtml(content, Html.FROM_HTML_MODE_LEGACY).toString()
+    } else {
+        Html.fromHtml(content).toString()
+    }
+}
```
Comment (Maxr1998) [inline app/src/main/java/org/jellyfin/mobile/utils/HtmlUtils.kt:12]: You should be able to replace this function with [`HtmlCompat.fromHtml()`](
https://developer.android.com/reference/androidx/core/text/HtmlCompat#fromHtml(java.lang.String,int)) called directly in `RemotePlayerService`. Note that the passed content string must be non-null, thus you should wrap the `fromHtml` call with a let, e.g. `title?.let { HtmlCompat.fromHtml(it, Html.FROM_HTML_MODE_LEGACY) }`.

**Review by Maxr1998 — CHANGES_REQUESTED** [review]
Thanks for investigating this! Your fix looks good, but I found you could replace your helper function with a compat function from AndroidX.

File: app/src/main/java/org/jellyfin/mobile/utils/HtmlUtils.kt (hunk 1 above)
Comment (colonelpopcorn) [reply app/src/main/java/org/jellyfin/mobile/utils/HtmlUtils.kt]: K, I made the fix as requested. For my edification, what happens if the null check returns true? Does the `it` lambda parameter fall through to null?

Comment (Maxr1998) [reply app/src/main/java/org/jellyfin/mobile/utils/HtmlUtils.kt]: If `title` is null, the let won't be executed and the whole statement evaluates to null.

**Review by Maxr1998 — APPROVED** [review]

**Comment by nielsvanvelzen** [comment]
The title is also used in some other places like the media session metadata. I'm not sure if we should fix the HTML in those places too.

**Comment by Maxr1998** [comment]
Are those shown somewhere where they wouldn't be decoded by the system?

**Comment by nielsvanvelzen** [comment]
Media session metadata can be read by any app and I don't think they expect HTML in the strings.

**Comment by Maxr1998** [comment]
Right. Should we still merge this already and have a different PR for the other issue?

**Comment by nielsvanvelzen** [comment]
Yeah that's fine

**Pull request merged**